### PR TITLE
Added flexible Travis-ci  build script.

### PR DESCRIPTION
**Pull request by ameyjadiye**
This enables ua to use any jdk against our code and not keeps us dependency on Travis to provide jdk

### CODE REVIEW

**Comment by chtompki** [comment]
I'll get to this soon.

**Comment by PascalSchumacher** [comment]
In my opinion this is overkill. I don't think commons-text needs to be tested against the eclipse java compiler and early access version of java 8 and 9. The script looks difficult to debug and maintain.

**Comment by chtompki** [comment]
Personally I'm a +0 to +1 on this, but @PascalSchumacher has a very valid point. I try to subscribe to the thought that there should be a high bar to sacrificing simplicity. So, with that in mind, should we take this up to the dev list to get other folks opinions? That work for you guys?

**Comment by PascalSchumacher** [comment]
+1 for moving this discussion to the mailing list

**Comment by PascalSchumacher** [comment]
Also note that currently this script only compiles and runs test.  Rat, clirr, findbugs and javadoc are not executed.

File: pom.xml [hunk 1]
```diff
@@ -171,6 +171,7 @@
               <exclude>src/site/resources/download_lang.cgi</exclude>
               <exclude>src/test/resources/stringEscapeUtilsTestData.txt</exclude>
               <exclude>src/site/resources/release-notes/RELEASE-NOTES-*.txt</exclude>
+              <exclude>.travis.sh</exclude>
```
Comment (chtompki) [inline pom.xml]: You could put an Apache License comment at the top of this file because it accommodates comments.

Comment (ameyjadiye) [inline pom.xml]: Sure, this was quick fix ;-) ... one more thing, do we need consent from jacoco using and modifying their file ?

Comment (chtompki) [inline pom.xml]: It looks like, with a quick read of their License.md, that we would minimally need to include their license at the end of ours, but I would have to defer to others for that.

Comment (ameyjadiye) [inline pom.xml]: which part of their license shall I add ? content of their ```License.md``` ? seems like they distribut code with eclipse license.

**Comment by ameyjadiye** [comment]
Because of low interest of community adapting this change i'm closing this pull request.

http://markmail.org/message/mn2or73hqsz6oxo2

**Pull request closed without merging**